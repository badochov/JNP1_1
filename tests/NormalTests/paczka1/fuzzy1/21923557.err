Error in line 5: a2YwZV8lm A558 30,8
Error in line 9: wqymUa3C A558 352,29
Error in line 12: wqymUa3C A558 240,3
Error in line 15: wqymUa3C S87 10,5
Error in line 21: a2YwZV8lm A418 274,6
Error in line 25: Y42 A418 16,1
Error in line 26: a2YwZV8lm A369 83,3
Error in line 11: 5C7tSCIcn A369 20,8
