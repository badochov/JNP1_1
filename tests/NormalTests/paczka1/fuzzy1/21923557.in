wqymUa3C A558 86,5
Y42 A558 88,2
?A418
5C7tSCIcn A369 43,4
a2YwZV8lm A558 30,8
Y42 A558 84,7
5C7tSCIcn A369 4,7
a2YwZV8lm A418 52,5
wqymUa3C A558 352,29
wqymUa3C A558 65,3
5C7tSCIcn A369 20,8
wqymUa3C A558 240,3
?A418
Y42 S87 24,4
wqymUa3C S87 10,5
E1gOfD6iP1j A558 11,5
Y42 S87 17,3
wqymUa3C A418 8,3
a2YwZV8lm A418 69,7

a2YwZV8lm A418 274,6

?A558
S87 A558 70,0
Y42 A418 16,1
a2YwZV8lm A369 83,3
Y42 S87 15,9
?
a2YwZV8lm A558 4,4
S87 A558 31,5
?A369
Y42 S87 52,3
wqymUa3C A418 53,6
r4GwrfGyHI A418 61,2
E1gOfD6iP1j A558 16,1
r4GwrfGyHI A418 193,4
?A558
wqymUa3C A418 93,3
?
5C7tSCIcn A558 24,7