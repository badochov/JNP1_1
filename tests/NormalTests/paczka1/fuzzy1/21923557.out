A558 24,7
5C7tSCIcn A 38,7
Y42 A 3,5 S 7,1
a2YwZV8lm A 17,2
wqymUa3C A 21,2
S87 7,1
A369 38,7
A418 17,2
A558 24,7
A369 38,7
A558 67,8
5C7tSCIcn A 38,7
E1gOfD6iP1j A 4,6
S87 A 38,5
Y42 A 3,5 S 43,5
a2YwZV8lm A 17,2
r4GwrfGyHI A 132,2
wqymUa3C A 66,5
S87 43,5
A369 38,7
A418 194,7
A558 67,8
